lerna.json

```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.0.0-beta.32",
+  "packages": [
+    "packages/*"
+  ],
+  "version": "5.0.0"
```

Comment:
So this is the version of happo, right? Does this automatically bump versions in all of the package.json files for us?